ports/serd/CMakeLists.txt

```
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.17)
+project(serd C)
+
+add_library(serd
+    src/byte_source.c
+    src/env.c
+    src/n3.c
+    src/node.c
+    src/reader.c
+    src/string.c
+    src/uri.c
+    src/writer.c
+)
+
+file(WRITE "${CMAKE_BINARY_DIR}/serd_config.h" "")
+
+target_include_directories(serd
+    PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}" "${CMAKE_BINARY_DIR}"
+)
+
+set_target_properties(serd PROPERTIES
+    C_STANDARD 99
+    C_STANDARD_REQUIRED ON
+)
+
+target_compile_definitions(serd PRIVATE SERD_INTERNAL)
+
+if(CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
+    target_compile_definitions(serd PRIVATE _CRT_SECURE_NO_WARNINGS)
+endif()

```

```suggestion
target_compile_definitions(serd PRIVATE SERD_INTERNAL _CRT_SECURE_NO_WARNINGS)

```

Shutting off those warnings unconditionally should be OK?